pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -59,62 +66,68 @@ func (l *PodInterface) Plug(iface *v1.Interface, network *v1.Network, domain *ap
 	precond.MustNotBeNil(domain)
 	initHandler()
 
-	driver, err := getBinding(iface)
+	driver, err := getBinding(iface, network, domain)
 	if err != nil {
 		return err
 	}
 
-	ifconf, err := getCachedInterface(iface.Name)
+	isExist, err := driver.loadCachedInterface(iface.Name)
 	if err != nil {
 		return err
 	}
 
-	interfaces := domain.Spec.Devices.Interfaces
-	podInterfaceNum, err := findInterfaceByName(interfaces, iface.Name)
-	if err != nil {
-		return err
-	}
-
-	if ifconf == nil {
+	if !isExist {
```

Comment:
hm. if we only care about file existence, why do we even bother to marshal / unmarshal?